samples/sdl_gamecontroller_event/main.c

```diff
@@ -0,0 +1,56 @@
+#include <xboxrt/debug.h>
+#include <pbkit/pbkit.h>
+#include <hal/video.h>
+#include <hal/xbox.h>
+#include <assert.h>
+#include <SDL.h>
+#include <stdbool.h>
+
+int main() {
+  XVideoSetMode(640, 480, 32, REFRESH_DEFAULT);
+  int ret = pb_init();
+  if (ret != 0) {
+    XSleep(2000);
+    return -1;
+  }
+
+  pb_show_debug_screen();
+
+  SDL_InitSubSystem(SDL_INIT_GAMECONTROLLER);
+
+  int gameControllerIndex = 0;
+
+  SDL_GameController *gameController = SDL_GameControllerOpen(gameControllerIndex);
+  if (gameController == NULL) {
+    debugPrint("Could not open GameController %i, please plug in a\n", gameControllerIndex);
+    debugPrint("compatible controller into port 1...\n");
```

Comment:
Where's the termination here? Otherwise the sample will be stuck in the while statement endless loop.

Should be similar to your sdl_gamecontroller sample.
https://github.com/XboxDev/nxdk/blob/a0f35be81b6856b953fbdc22e557c4097405f335/samples/sdl_gamecontroller/main.c#L24-L29

Reply:
Good catch, fixed now!

Reply:
New solution isn't ideal either (will close even if controller from unused port is disconnected?). Anyhow, can be a separate discussion / review comment.